README-ZH.md

```diff
@@ -87,6 +89,13 @@
 
 Flame的官方网站(其中包含文档), 请[点击此处](https://flame-engine.org/)
 
+
+## 其他中文社区资源
+
+Flame中文网(社区版), 包含官方的中文翻译, 请 [点击此处](https://flame-cn.com/)
+
+Flame中文论坛(社区版), 请 [点击此处](https://bbs.flame-cn.com/)
```

Comment:
can you explain better what is this? I would rather not add one-off links that are not present in the other versions, maybe you can try submitting an english version to flame-awesome instead?

Reply:
Ok, I just move those lines

Reply:
Shouldn't this represent a one-to-one translation of the original readme?
I can still see the links in the PR, what do you mean with that you moved them?
Like Luan says, maybe better to move them to flame-awesome, so that this file will be kept consistent over the different languages.